src/components/datasets-list/datasets-item-header/index.tsx

```diff
@@ -0,0 +1,36 @@
+'use client';
+import Info from '@/components/datasets-list/datasets-item-info';
+import Icon from '@/components/icon/component';
+import DOWNLOAD_SVG from 'svgs/ui/download.svg?sprite';
+import SHARE_SVG from 'svgs/ui/share.svg?sprite';
+export const DatasetsItemHeader = ({
+  layer_id,
+  title,
+  downloadUrlBase,
+}: {
+  layer_id: string;
+  title: string;
+  downloadUrlBase?: string;
+}) => {
```

Comment:
I prefer types in a variable, I think it will scale better, and it's reusable. But this is also ok.